r/NAMESPACE

```diff
@@ -85,6 +85,7 @@ S3method(write_feather_RecordBatch,"arrow::io::OutputStream")
 S3method(write_feather_RecordBatch,character)
 S3method(write_feather_RecordBatch,default)
 S3method(write_feather_RecordBatch,fs_path)
+export()
```

Comment:
not sure where this empty `export()` comes from ?

Reply:
reverted, you might need to check for winbuilder warnings for re-exports.